rectools/visuals/metrics_app.py

```diff
@@ -20,6 +20,7 @@
 import plotly.express as px
 import plotly.graph_objs as go
 from IPython.display import display
+from packaging.version import Version
```

Comment:
If you're explicitly using a package outside of the standard library, please add it to the list of dependencies

Reply:
(Read next comment before fixing this one)

Reply:
Outdated, because we added nbformat in project dependencies